solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1122,6 +1131,7 @@ impl pallet_beefy::Config for Runtime {
     type KeyOwnerProof = <Historical as KeyOwnerProofSystem<(KeyTypeId, BeefyId)>>::Proof;
     type EquivocationReportSystem =
         pallet_beefy::EquivocationReportSystem<Self, Offences, Historical, ReportLongevity>;
+    type AncestryHelper = MmrLeaf;
```

Comment:
wow what is this haha

Reply:
Something used for the `report_fork_voting` extrinsic. I'm not familiar with beefy

Reply:
Maybe not for this particular PR, but beefy is not really needed for solo chain and we can probably remove it. 

It maintains merkle mountain range of block hashes and can be used while bridging with other blockchains like ethereum for efficient verification of state of the blockchain.